Sources/KlaviyoSwift/KlaviyoModels.swift

```diff
@@ -194,6 +211,12 @@ extension Event.EventName {
     var value: String {
         switch self {
         case .OpenedPush: return "$opened_push"
+        case .OpenedAppMetric: return "Opened App"
+        case .ViewedProductMetric: return "Viewed Product"
+        case .AddedToCartMetric: return "Added to Cart"
+        case .StartedCheckoutMetric: return "Started Checkout"
+        case let .CustomEvent(value): return "\(value)"
+
         case .ViewedProduct: return "$viewed_product"
```

Comment:
Should these go away?

Reply:
next release

Reply:
Ahh sorry got confused...thought I was looking at that PR!